apps/api/src/utils.ts

```
@@ -1,9 +1,23 @@
 
+/**
+ * Throws an HTTP 404 Not Found exception with a custom message.
+ *
+ * @param msg - The error message to include in the response.
+ *
+ * @throws {HTTPException} Always thrown with status 404 and the provided {@link msg}.
+ */
 export function notFound(c: Context, msg: string) {
-  return c.json({ error: msg }, 404);
+  throw new HTTPException(404, { message: msg });
+  return c.json({ error: msg, success: false }, 404);
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)